rust_src/src/eval.rs

```diff
@@ -1301,4 +1302,36 @@ pub fn signal(error_symbol: LispObject, data: LispObject) -> ! {
     }
 }
 
+/// Regain control when an error is signaled.
+/// Executes BODYFORM and returns its value if no error happens.
+/// Each element of HANDLERS looks like (CONDITION-NAME BODY...)
+/// where the BODY is made of Lisp expressions.
+///
+/// A handler is applicable to an error
+/// if CONDITION-NAME is one of the error's condition names.
+/// If an error happens, the first applicable handler is run.
+///
+/// The car of a handler may be a list of condition names instead of a
+/// single condition name; then it handles all of them.  If the special
+/// condition name `debug' is present in this list, it allows another
+/// condition in the list to run the debugger if `debug-on-error' and the
+/// other usual mechanisms says it should (otherwise, `condition-case'
+/// suppresses the debugger).
+///
+/// When a handler handles an error, control returns to the `condition-case'
+/// and it executes the handler's BODY...
+/// with VAR bound to (ERROR-SYMBOL . SIGNAL-DATA) from the error.
+/// \(If VAR is nil, the handler can't access that information.)
+/// Then the value of the last BODY form is returned from the `condition-case'
+/// expression.
+///
+/// See also the function `signal' for more info.
+/// usage: (condition-case VAR BODYFORM &rest HANDLERS)
+#[lisp_fn(min = "2", unevalled = "true")]
+pub fn condition_case(args: LispCons) -> LispObject {
+    let (var, consq) = args.into();
+    let (bodyform, handlers_) = consq.into();
```

Comment:
Why the underscore on handlers? I do not see a name conflict.
```suggestion
    let (bodyform, handlers) = consq.into();
```